sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_multiapi.py

```diff
@@ -0,0 +1,46 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from enum import Enum
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Union
+
+
+class ApiVersion(str, Enum):
+    """Key Vault API versions supported by this package"""
```

Comment:
```suggestion
    """Text Analytics API versions supported by this package"""
```

Reply:
Oh no. Thanks for finding that, and I guess you know where I copied all the code from